eng/pipelines/spec-gen-sdk.yml

```diff
@@ -19,17 +19,144 @@ parameters:
   - name: SkipPullRequestCreation
     type: boolean
     default: false
-    
-trigger: none
-pr: none
-
-extends:
-  template: /eng/common/pipelines/templates/jobs/archetype-spec-gen-sdk.yml
-  parameters:
-    SpecRepoUrl: 'https://github.com/$(Build.Repository.Name)'
-    SdkRepoUrl: $(SdkRepoUrl)
-    SpecRepoCommit: ${{ parameters.SpecRepoCommit }}
-    SdkRepoCommit: ${{ parameters.SdkRepoCommit }}
-    ConfigType: ${{ parameters.ConfigType }}
-    ConfigPath: ${{ parameters.ConfigPath }}
-    SkipPullRequestCreation: ${{ parameters.SkipPullRequestCreation }}
\ No newline at end of file
+
+trigger:
+  branches:
+    include:
+      - user/raych1/support-spec-pr
```

Comment:
These triggers settings are for testing purpose, and they will be updated before rolling out the pipeline.